src/pages/ESNMembersTab/EsnersSheet.tsx

```diff
@@ -97,35 +115,46 @@ function EsnersSheet() {
 
     const boardMembers = esners
         .filter(member => member.memberType === MEMBER_BOARD)
-        .map((data: ESNer) => <Member key={data.id} data={data} />);
+        .map((data: ESNer) => <Member key={data.id} data={data} openModal={openModal} />);
 
     const coordinators = esners
         .filter(member => member.memberType === MEMBER_COORDINATOR)
-        .map((data: ESNer) => <Member key={data.id} data={data} />);
+        .map((data: ESNer) => <Member key={data.id} data={data} openModal={openModal} />);
 
     const ordinaryMembers = esners
         .filter(member => member.memberType === MEMBER_ORDINARY)
-        .map((data: ESNer) => <Member key={data.id} data={data} />);
+        .map((data: ESNer) => <Member key={data.id} data={data} openModal={openModal} />);
 
     if (esners === []) return <div> NO ESNERS FOUND </div>;
     return (
-        <IonList  className="background-white-opacity">
-            <IonItemGroup className="div-box-members">
-
-                <IonLabel className="background-white ion-text-center box-members" color="blackNormal"><img className="hamburger-menu" src="assets/button.png" alt="Menu icon"/> Board</IonLabel>
-                {boardMembers}
-
-            </IonItemGroup>
-
-            <IonItemGroup>
-                <IonLabel className="background-white ion-text-center box-members" color="blackNormal"><img className="hamburger-menu" src="assets/button.png" alt="Menu icon"/>Coordinators</IonLabel>
-                {coordinators}
-            </IonItemGroup>
-            <IonItemGroup>
-                <IonLabel className="background-white ion-text-center box-members" color="blackNormal"><img className="hamburger-menu" src="assets/button.png" alt="Menu icon"/>Ordinary Members</IonLabel>
-                {ordinaryMembers}
-            </IonItemGroup>
-        </IonList>
+        <>
```

Comment:
Why do we need this here?

Reply:
In react you can't return multiple elemets. It always needs to be one `div` or sth similar that wrapps everything.

<> </> is a way to do it without creating another div. React treats this as one element, but later when page is rendering engine omits this element.

More: 
https://reactjs.org/docs/fragments.html